xml/CRSChannelCode.xml

```diff
@@ -586,11 +586,18 @@
                 <narrative xml:lang="fr">Alliance pour les villes</narrative>
             </name>
         </codelist-item>
+        <codelist-item status="active" activation-date="2019-11-10">
```

Comment:
The `activation-date` here (and elsewhere) is set to the day the code ran. That’s probably a bit silly – it would be better to update these with something more accurate.